cmd/app/internal/service/tts-stt/tts-stt.py

```
@@ -91,50 +142,176 @@ async def speech_to_text(file: UploadFile):
+# -----------------------------------
+# Streaming TTS endpoint (for real-time feel)
+# -----------------------------------
+@app.post("/tts-stream")
+async def text_to_speech_stream(text: str = Form(...)):
+    """Stream audio as it's generated for better perceived performance"""
+
+    async def audio_generator():
+        text_chunks = split_text_smartly(text, max_length=50)
+
+        for chunk in text_chunks:
+            chunk_wav = f"/tmp/{uuid.uuid4()}_stream.wav"
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)